.github/workflows/terraform-apply-actions

```diff
@@ -0,0 +1,41 @@
+name: Terraform Workflow
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
```

Comment:
Nice 👌 
<img width="880" alt="image" src="https://github.com/user-attachments/assets/e396d724-2e54-44b9-9e2f-4301da607ab9">

Are you able to test this by running this on your branch (removing hardcoded main on L6)? Just to be sure that this works?

Reply:
This is a limitation workflow dispatch works on main branch only regardless of which branch we declared in workflow

Reply:
But did my testing here https://github.com/samurato/gh-actions-test